### PR TITLE
feat(helm): support labels in values.yaml

### PR DESCRIPTION
**Pull request by razvan**
Part of: https://github.com/stackabletech/issues/issues/501

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,16 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Helm: support labels in values.yaml ([#201]).
```
Comment (Techassi) [inline CHANGELOG.md:9]: note: This PR number needs to be changed to `203`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,11 +4,16 @@ All notable changes to this project will be documented in this file.
...
+
 ### Fixed
 
 - Respect `--watch-namespace` CLI argument ([#193]).
 
 [#193]: https://github.com/stackabletech/commons-operator/pull/193
+[#201]: https://github.com/stackabletech/commons-operator/pull/201
```
Comment (Techassi) [inline CHANGELOG.md:16]: note: This link also needs to be changed.

**Review by Techassi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (razvan) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (razvan) [reply CHANGELOG.md]: done

**Review by Techassi — APPROVED** [review]

**Pull request merged**